packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -228,23 +231,26 @@ export async function pollDeploymentStatus(deployCtx: DeployContext) {
             currentStatus[operation.resourceName] = operation.status;
             if (!polledOperations.includes(operation.resourceName)) {
               polledOperations.push(operation.resourceName);
-              let client = deployCtx.client;
-              if (operation.subscriptionId !== deployCtx.client.subscriptionId) {
-                const azureToken =
-                  await deployCtx.azureAccountProvider?.getAccountCredentialAsync();
-                client = new ResourceManagementClient(azureToken!, operation.subscriptionId);
-              }
-
-              const subOperations = await client.deploymentOperations.list(
-                operation.resourceGroupName,
-                operation.resourceName
-              );
-              subOperations.forEach((sub) => {
-                const subOperation = getRequiredOperation(sub, deployCtx);
-                if (subOperation) {
-                  currentStatus[subOperation.resourceName] = subOperation.status;
+              // get sub operations when resource type is deployments.
+              if (operation.resourceType === ConstantString.DeploymentResourceType) {
```

Comment:
if the resouceType is not deployment, we will not collect infomation?

Reply:
When it is not a deployment, it has no sub operation. But we will show it in output as well.